torch/distributed/tensor/_op_schema.py

```diff
@@ -512,34 +517,7 @@ def __eq__(self, other: object) -> bool:
         if len(self.args_schema) != len(other.args_schema):
             return False
 
-        # compare each element and early return if any of them is different
```

Comment:
although the code here isn't directly moved into comparison key, i confirmed that the args_to_hash and kwargs_to_hash members are equivalent to the logic below, this LGTM